backends/arm/test/ops/test_linear.py

```diff
@@ -172,9 +176,7 @@ def _test_linear_tosa_u55_BI_pipeline(
             .to_executorch()
             .serialize()
         )
-
-        if common.is_option_enabled("corstone300"):
```

Comment:
I am not sure if I am following the pattern where we enable/disable these. Is this operator specific? I would rather tie it to the ip i.e. something like this, hypothetically

```
if common.is_option_enabled("corstone300") and compile_spec.ip == "u55":
```

Reply:
the `common.is_option_enabled("corstone300")` only says if the FVP is installed on the target and can be run. 
Right now it's only added for a few operators, but due to the time it adds to CI we probably want to move away from enabling it for all operator unit tests all the time.

Reply:
IIUC this FVP is u55 specific while this function can also get u85 compile spec and vela/et can produce u85 binary. I was trying to highlight that the choice of fvp 300 vs 320 should be tied to the compile spec somehow.